addons/website/__manifest__.py

```diff
@@ -133,10 +133,12 @@
         ],
         'web.assets_frontend_minimal': [
             'website/static/src/js/content/inject_dom.js',
+            'website/static/src/js/content/auto_hide_menu.js',
```

Comment:
I don't think your commit message make a lot of sense for this one... you explain that the items were wrapper on multiple lines and that this replaces the behavior by the new version of initAutoMoreMenu... but I don't think you changed that?